### ISSUE TITLE
importing css from vendor/assets/stylesheets fails

### ISSUE THREAD

**Issue by timbleck** [hagenburger/livingstyleguide#98]
We're having an application.sass and trying to `@import` a plain css (`vendor/assets/stylesheets/jquery.bxslider.css`) file from `vendor/assets/stylesheets`.

Using the rails asset pipeline this works fine but within the styleguide we get an error that the file could not be found.

    Sass::SyntaxError: File to import not found or unreadable: jquery.bxslider.
    Load paths:
      Compass::SpriteImporter
      /.../app/assets/images
      /.../app/assets/javascripts
      /.../app/assets/stylesheets
      /.../vendor/assets/javascripts
      /.../vendor/assets/stylesheets

Our application.sass

    ...
    @import jquery.bxslider
    ...

Converting the plain css file to sass works but as we wan't to just import external plugins this is not the best solution.

**Comment by jmcnevin** [comment]
We're experiencing this as well.

```
livingstyleguide (1.2.2)
sass-rails (5.0.1)
jquery-ui-rails (= 5.0.3)
```

In a stylesheet, I have:

```
@import 'jquery-ui/autocomplete';
```

Error:

```
File to import not found or unreadable: jquery-ui/autocomplete
```

**Comment by hagenburger** [comment]
I’m going to release 2.0.0.alpha.2 asap. A completely reworked load path might be part of 2.0.0.alpha.3/4—depending how complicated this gets. Right now for 2.0.0.alpha.1 all paths must be relative from `Rails.root`. Of course this might only work on one machine due to different paths for Gems.

**Comment by hagenburger** [comment]
I had a deep look into `sprockets`, `sprockets-rails`, `sass-rails`, `tilt`, and `middleman` which led into a complete refactoring. Having the new code, LSG should render Sass exactly the same way Rails/Middleman does and provide access to all helpers and load paths.

This will be available from 2.0.0.alpha.3 on.
